source/ways-of-working/asking-for-help/index.html.md.erb

```diff
@@ -26,6 +26,7 @@ These Slack channels are good places to ask for help:
 - [#cloud-native](https://hmcts-reform.slack.com/messages/cloud-native) is a community channel where people can discuss best practices and ask general questions
 - [#platops-help](https://hmcts-reform.slack.com/messages/platops-help) is for support requests to the Platform Operations team
 - [#idam-support-cft](https://hmcts-reform.slack.com/messages/idam-support-cft) for support and questions related to IDAM
+- [#sds-cloud-native](https://hmcts-reform.slack.com/messages/sds-cloud-native) ask general questions about SDS platform 
```

Comment:
I would copy similar wording to #cloud-native but have a reference this is for sds platform